packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -619,8 +622,13 @@ class PageView extends StatefulWidget {
   final bool pageSnapping;
 
   /// Called whenever the page in the center of the viewport changes.
+  ///
+  /// It is called before [pageDidChange], in the middle of the scroll animation.
   final ValueChanged<int> onPageChanged;
-
+  /// Called whenever the page has changed and the scroll animation has stopped.
```

Comment:
Nit: Please place a line break between each of these properties.
```suggestion

  /// Called whenever the page has changed and the scroll animation has stopped.
```